build-patched-kao-kcmo-images.sh

```diff
@@ -95,7 +95,10 @@ function patch_and_push_image() {
     git checkout --track origin/rhaos-${OCP_VERSION}-rhel-8
     git merge --no-ff -m "Merge commit ${vcs_ref} into rhaos-${OCP_VERSION}-rhel-8" -m "MaxFileSize: 104857600" ${vcs_ref}
     git push origin HEAD
-    rhpkg container-build --target crc-1-rhel-8-candidate
+    # If brew build already exist for the release don't rebuild it again
+    if ! brew list-builds --package=crc-${image_name}-container | grep -q ${release}; then
```

Comment:
`brew buildinfo crc-${image_name}-container-${release}` instead of `list-builds`+`grep`? Might be better to return early before the `rhpkg clone` when we don't need to build?

Reply:
@cfergeau `rhpkg clone` is needed because that is what dnsmasq/router images consume for updating the base image.

Reply:
> brew buildinfo crc-${image_name}-container-${release} instead of list-builds+grep? 

Sure, this we can update.

Reply:
> @cfergeau rhpkg clone is needed because that is what dnsmasq/router images consume for updating the base image.

If we don't need to rebuild crc-cluster-kube-apiserver-operator, this means in particular its base image won't have changed, so we don't need to rebuild the dnsmasq/router images, the ones we have are already up to date.

Reply:
> If we don't need to rebuild crc-cluster-kube-apiserver-operator, this means in particular its base image won't have changed, so we don't need to rebuild the dnsmasq/router images, the ones we have are already up to date.

Sure, in that case we first check if the respective directory exist and if not then not need to build `dnsmasq/router` images.

Reply:
And in that case we can push that change before rhpkg so we don't need to pull it.